### ISSUE TITLE
Error alerts in Power Apps

### ISSUE THREAD

**Issue by dClem12** [OfficeDev/microsoft-teams-apps-attendance#18]
Hi all, 
My Power Apps page shows several errors when trying to test and edit the app. Any suggestions? I think it is to do with data types but I have checked these in SharePoint, deleted them and recreated again. 
![image](https://user-images.githubusercontent.com/63758358/79400040-5b4a3c80-7fc8-11ea-9ad4-8cebc4957a46.png)

**Comment by abbodh** [comment]
hi @dClem12 , can you please confirm that SharePoint connection is set up correctly with app? 

https://github.com/OfficeDev/microsoft-teams-apps-attendance/wiki/Deployment-guide#step-6-edit-data-source 

Ideally only one error (for user profile photo) would be shown during Edit mode in Power Apps if there is any failure.

**Comment by Prachii2020** [comment]
Yes you can check for the share-point  connection is working fine. Apart from this the error for user
profile photo is a power app issue  For loading images(profile pic) it is taking time and hence it shows that error. It does not affect any functionality and can be ignored.
